src/v2/Apps/Artwork/Components/SubmittedOrderModal/SubmittedOrderModal.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Message, ModalDialog, Text } from "@artsy/palette"
+import { FC, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  getStatusCopy,
+  continueToInboxText,
+} from "v2/Apps/Order/Utils/getStatusCopy"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { SubmittedOrderModal_order } from "v2/__generated__/SubmittedOrderModal_order.graphql"
+
+interface SubmittedOrderModalProps {
+  order: Pick<SubmittedOrderModal_order, "mode" | "stateExpiresAt">
```

Comment:
I'm using `Pick` because I have this error with `$fragmentReft`, but it should not be necessary:
```
Property '' $fragmentRefs'' is missing in type 'SubmittedOrderModal_order' but required in type '_FragmentRefs<"SubmittedOrderModal_order">'.ts(2741)
index.d.ts(215, 5): '' $fragmentRefs'' is declared here.
SubmittedOrderModal.tsx(12, 3): The expected type comes from property 'order' which is declared here on type 'IntrinsicAttributes & MappedFragmentProps<Pick<SubmittedOrderModalProps, "order">> & { componentRef?: ((ref: any) => void) | undefined; } & { ...; }'
```

Reply:
You should stick to the relay generated type from the fragment query, the prop, in this case, should contain `" $refType": "SubmittedOrderModal_order"` and not only the 2 other props.
Probably is something with your VSCode, consider restarting it and should solve the issue. Another thing you can also run is `yarn relay:clean` to clean some weird Relay stuff from here.

Reply:
I tried here I no error popped out.
There is no relation to another fragment type such as `Review_order`, in your case you're creating the fragment `SubmittedOrderModal_order` from graphql call, and relay compiler generates that type which should be used in the relay fragment prop of the component, no other way around.